src/server/utils/jsUtils.js

```diff
@@ -197,10 +197,20 @@ function retryAsync(promiseFunc, maxTries, delay, iteration) {
     });
 }
 
+function stripErrorColon(err) {
```

Comment:
Hmmm... I'm not sure what you mean. This is for processing errors that come from elsewhere and often have "Error:" at the start.